src/parameterized/gamma.jl

```diff
@@ -0,0 +1,27 @@
+# Gamma distribution
+
+export Gamma
+
+@parameterized Gamma(α,β) ≪ Lebesgue(ℝ₊)
```

Comment:
Do you see a reason not to use the Distributions parameterization for the default? We can have both of course

Reply:
I spontaneously used the one I was familiar with, but you're right, for consistency sticking to Distributions is better style

Reply:
Ok let's have the Distributions form as the default, and also add the alpha,beta parameterization